modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -572,9 +572,10 @@ static void setBlob(InferenceEngine::InferRequest& req,
 static void setROIBlob(InferenceEngine::InferRequest& req,
                        const std::string&             layer_name,
                        const IE::Blob::Ptr&           blob,
-                       const cv::Rect &roi,
+                       const cv::Rect                 &roi,
                        const IECallContext&           ctx) {
-    if (ctx.uu.params.device_id.find("GPU") != std::string::npos) {
+    if (ctx.uu.params.device_id.find("GPU") != std::string::npos &&
```

Comment:
Please throw error in that case, silently ignoring ROI just creating blob on the full image is confusing behavior

Reply:
ok, let's through then